test/plugin/scenarios/rocketmq-5-grpc-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+  - serviceName: rocketmq-5-grpc-scenario
+    segmentSize: ge 2
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: HEAD:/case/healthCheck
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 14
+            isError: false
+            spanType: Entry
+            skipAnalysis: 'false'
+            tags:
+              - { key: url, value: 'http://localhost:8080/rocketmq-5-grpc-scenario/case/healthCheck' }
+              - { key: http.method, value: HEAD }
+              - { key: http.status_code, value: '200' }
+      - segmentId: not null
+        spans:
+          - operationName: RocketMQ/TopicTest/Producer
+            parentSpanId: 0
+            spanId: 1
+            spanLayer: MQ
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 38
+            isError: false
+            spanType: Exit
+            peer: not null
+            tags:
+              - {key: mq.broker, value: not null}
+              - {key: mq.topic, value: TopicTest}
+              - {key: mq.message.keys, value: KeyA}
+              - {key: mq.message.tags, value: TagA}
+              - {key: mq.message.id, value: not null}
+            skipAnalysis: 'false'
+          - operationName: GET:/case/rocketmq-5-grpc-scenario
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 14
+            isError: false
+            spanType: Entry
+            peer: ''
+            tags:
+              - {key: url, value: 'http://localhost:8080/rocketmq-5-grpc-scenario/case/rocketmq-5-grpc-scenario'}
+              - {key: http.method, value: GET}
+              - {key: http.status_code, value: '200'}
+            skipAnalysis: 'false'
+      - segmentId: not null
+        spans:
+          - operationName: RocketMQ/TopicTest/Consumer
```

Comment:
For the consumer, where is the `ref`? This entry span should have a ref pointing to the provider's span. Otherwise, you actually don't verify the tracing is continuing across processes.

Reply:
i fix it